osuosl/climesync
# Made activities field optional if project has a default activity

fixes issue #76 

## Changes in this PR.


- [X] If the project associated with the time being created has a default activity then the `activities` field is optional. Otherwise, it's a required field.

## Testing this PR.


1. Run `nosetests`
2. Run climesync, sign in, and find a project that has a default activity (using `gp`)
3. Create a time with `ct` and use the slug of the project you just found
4. See that `activities` is optional
5. Go back and create another time for a project that **doesn't** have a default activity
6. See that `activities` is now required

@osuosl/devs
